examples/standalone/library-import-cjs/README.md

```diff
@@ -0,0 +1,39 @@
+# Bundle Analyzer Library Import (CJS) Example
+
+This directory includes an example calling the bundle-analyzer exported library for bundle analysis.
+
+This example runs in a CJS environment.
+
+To run:
+
+```
+# build the library
+cd packages/bundle-analyzer
+pnpm install
+pnpm run build
+
+# create an example build to analyze
+cd examples/bundle-analyzer/cli
+pnpm install
+pnpm run build
+
+# run the node program
+cd examples/bundle-analyzer/library-import
+pnpm install
+pnpm run start
+```
+
+This will call the imported function. Note a Codecov API server (e.g., that at `test-api`) needs to be running for any uploads (i.e., not dry-run) to succeed.
```

Comment:
nit: `e.g., that at 'test-api'` should `at` be `a`?

Reply:
Thanks! Updated up this sentence a little since it may have been confusing in its prior state-
`This will call the imported function that creates and uploads (or dry-runs) the bundle stats report.`